config.json

```diff
@@ -964,7 +964,38 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+    {
+      "title": "Functional",
+      "content": "Multi-clause functions with pattern matching and guards are the building blocks of Erlang code.",
+      "icon": "functional"
+    },
+    {
+      "title": "Dynamically typed",
+      "content": "Erlang has no compile-time type checks, favoring run-time pattern matching.",
+      "icon": "dynamically-typed"
+    },
+    {
+      "title": "Immutable",
+      "content": "All data in Erlang is immutable, allowing for safer and easier-to-reason-about concurrency.",
+      "icon": "immutable"
+    },
+    {
+      "title": "Concurrent",
+      "content": "Erlang uses the actor model - shared-nothing concurrency via message passing.",
+      "icon": "concurrency"
+    },
+    {
+      "title": "Fault tolerant",
+      "content": "Erlang runs in a VM known for running low-latency, distributed and fault-tolerant systems.",
+      "icon": "safe"
+    },
+    {
+      "title": "Flexible package manager",
```

Comment:
I think we can remove this one.

Rebar is not as flexible as mix is, and it **not** part of the language or its standard toolchain.

Reply:
🤔  need to come up with a different one then. It has to be exactly 6.

Reply:
@ErikSchierboom @iHiD is this a true limitation?

Reply:
It is. You could look at more tracks for inspiration if you'd like

Reply:
Maybe something about being "battle-tested" (but in nicer, not war-related wording) about Erlang's long history of working well? Or something about Erlang being "influential" and inspiring other languages being created based on it? Or about the community?

Reply:
How about "massively scalable"? That sort of touches on "battle tested" and also widens it sligthly.

This is from Erlang's website and so that scalability seems to be the first/loudest feature of the language:
> Erlang is a programming language used to build massively scalable soft real-time systems with requirements on high availability. Some of its uses are in telecoms, banking, e-commerce, computer telephony and instant messaging. Erlang's runtime system has built-in support for concurrency, distribution and fault tolerance.